hotbar.coffee

```diff
@@ -92,16 +94,28 @@ class InventoryHotbarClient extends InventoryHotbarCommon
   enable: () ->
     @inventoryWindow.container.style.visibility = ''
     @onSlots = {}
+    @console.on 'open', =>
```

Comment:
Note to self: requires https://github.com/deathcap/voxel-console/pull/3. I'll review that PR first